workload/master_config.yaml

```diff
@@ -0,0 +1,43 @@
+s3api:
```

Comment:
This s3api should not be similar to any operation instead can we have a different name something like basic_io?

Reply:
Agreed, Let me out in comment, we will think over and consider once will start implementing master_config.